src/lib/utils/skip_test.cpp

```diff
@@ -0,0 +1,44 @@
+#define UNW_LOCAL_ONLY
+#include <cxxabi.h>
+#include <gtest/gtest.h>
+#include <libunwind.h>
+
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+[[noreturn]] void skip_test() {
+  // The idea is to unwind the stack, search for the googletest method that called the test and reset the instruction
```

Comment:
oh please, no. Is there really no however ugly way to solve this?

Reply:
e.g. a AVAILABLE_ENCODINGS global variable that contains all available encoding that tests can use to instantiate themselves?